src/utils/FSHLogger.ts

```diff
@@ -17,7 +17,22 @@ const withLocation = format(info => {
   return info;
 });
 
+const incrementCounts = format(info => {
+  if (info.level === 'info') stats.numInfo++;
+  if (info.level === 'warn') stats.numWarn++;
+  if (info.level === 'error') stats.numError++;
```

Comment:
I think this would be bit cleaner as a `switch` statement on `info.level` -- but if you prefer `if`, then I'd use `else if` to be more efficient.

Reply:
Sure, I can change it to a `switch`.